apisix/core/config_etcd.lua

```diff
@@ -523,6 +523,7 @@ local function _automatic_fetch(premature, self)
     end
 
     if not (health_check.conf and health_check.conf.shm_name) then
+        -- used for worker processes to synchronize fetch the configuration
```

Comment:
`synchronize` is already a verb. We usually say `synchronize configuration` but not `synchronously fetch configuration`.

Reply:
updated